lib/net.js

```diff
@@ -1326,84 +1326,70 @@ function listen(self, address, port, addressType, backlog, fd, exclusive) {
 
 
 Server.prototype.listen = function() {
-  var self = this;
+  var args = new Array(arguments.length);
+  for (var i = 0; i < arguments.length; i++)
+    args[i] = arguments[i];
+  args = normalizeConnectArgs(args);
```

Comment:
Can you please rename this to `normalizeArgs`, and add a quick comment to mention that it is used on both places?